homeassistant/components/daikin.py

```diff
@@ -0,0 +1,294 @@
+"""
+Platform for the Daikin AC.
+
+For more details about this component, please refer to the documentation
+https://home-assistant.io/components/daikin/
+"""
+import logging
+import time
+from socket import timeout
+from threading import Lock
+
+import voluptuous as vol
+
+from homeassistant.components.climate import (
```

Comment:
This feels wrong. This (and everything using these climate constants) should belong into the climate platform.

Reply:
The DaikinEntity is the main class using them and it implements common methods for both climate and sensors. Where should this class be placed in HA since I would not want to duplicate the code in both climate component and sensor class ?

Should I place it back in the climate.daikin class and share it through hass.data[DOMAIN][host] with the sensors or do u have other suggestion ?

Reply:
@tinloaf I've moved the code back in the climate component

Reply:
I'll give it a look, but that sounds good!